services/movie.py

```diff
@@ -0,0 +1,32 @@
+from typing import List
+from db.models import Movie
+from django.db.models.query import QuerySet
+
+
+def get_movies(genres_ids: list = None,
+               actors_ids: List[int] = None) -> QuerySet:
+    queryset = Movie.objects.all()
+    if genres_ids is not None:

```

Comment:
```suggestion
    if not genres_ids :

```

Reply:
the same in another case

Reply:
in case  "if not genres_ids :"  gives an error  TypeError: 'NoneType' object is not iterable